ci/release/update-version.sh

```diff
@@ -53,6 +53,7 @@ DEPENDENCIES=(
   librmm
   rapids-dask-dependency
   rmm
+  pylibcudf
```

Comment:
Can you move this up so the list if alphabetical?

Reply:
Sure